src/less/cta-button/stories/dimensions.stories.js

```diff
@@ -11,21 +11,32 @@ export const large = () => `
 </a>
 `;
 
-export const largeTruncated = () => `
-<a class="cta-btn cta-btn--large-truncated" href="http://www.ebay.com">
+export const largeFixedWidthTruncated = () => `
+<a style="width: 200px;" class="cta-btn cta-btn--large-truncated" href="http://www.ebay.com">
     <span class="cta-btn__cell">
-        <span>Link</span>
+        <span>Link with a lot of text that should wrap</span>
         <svg class="icon icon--cta" focusable="false" height="8" width="8" aria-hidden="true">
             <use xlink:href="#icon-cta"></use>
         </svg>
     </span>
 </a>
 `;
 
-export const largeFixedHeight = () => `
-<a class="cta-btn cta-btn--large-fixed-height" href="http://www.ebay.com">
+export const largeFixedWidthAndHeight = () => `
```

Comment:
I don't think we want this specific scenario. I can't imagine a use case in which we'd want to use this UI: 
<img width="549" alt="image" src="https://user-images.githubusercontent.com/1675667/201381917-1a44432f-c301-4fa8-a68c-33efb8acf12c.png">

Reply:
Agree 100%. Added as button has it too for consistency.
https://opensource.ebay.com/skin/storybook/?path=/story/button-dimensions--fixed-width-and-height

Will remove this.